modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -42,9 +39,8 @@ spec:
     secretName: example-com-tls-certificate
 ----
 +
-<1> The `route.openshift.io/termination` annotation can be used to configure the `spec.tls.termination` field of the `Route`
-as `Ingress` has no field for this. The accepted values are `edge`, `passthrough` and `reencrypt`. All
-other values are silently ignored. When the annotation value is unset, `edge` is the default route. The TLS certificate details must be defined in the template file to implement the default edge route.
+<1> The `route.openshift.io/termination` annotation can be used to configure the `spec.tls.termination` field of the `Route` as `Ingress` has no field for this. The accepted values are `edge`, `passthrough` and `reencrypt`. All other values are silently ignored. When the annotation value is unset, `edge` is the default route. The TLS certificate details must be defined in the template file to implement the default edge route.
+<2> When working with an Ingress object, you must specify an explicit host name, unlike when working with routes. You can use the `<host_name>.<cluster_ingress_domain>` syntax, for example `apps.openshiftdemos.com`, to take advantage of the cluster's `*.<cluster_ingress_domain>` wildcard DNS record and serving certificate. Otherwise, you must ensure that there is a DNS record for the chosen host name.
 
 .. If you specify the `passthrough` value in the `route.openshift.io/termination` annotation, set `path` to `''` and `pathType` to `ImplementationSpecific` in the spec:
```

Comment:
In the preview, this line looks to be nested within call out 2 of the previous step. You might need to add a `+` between this line and the preceding call out.